vignettes/snomedizer.Rmd

```diff
@@ -0,0 +1,373 @@
+date: '`r format(Sys.Date(), "%d %B %Y")`'
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    always_allow_html: yes
+    df_print: tibble
+vignette: >
+  %\VignetteIndexEntry{snomedizer: R Interface to the SNOMED CT Terminology Server REST API}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(dplyr)
+library(snomedizer)
+```
+
+# Overview
+
+## What is SNOMED CT?
+
+SNOMED CT is an international ontology of clinical terms. You can think of it as an encyclopedia on all things that are talked about in medicine, for instance: anatomical structures, diagnoses, medications, tests,  surgical procedures, pathogens such as SARS-CoV-2 or *Candidida auris*, or even clinical findings, such as a high temperature.
+
+SNOMED CT is used mainly in the UK:
+
+* to record information about patient care in general practice and hospitals
+* to analyse electronic health records, in particular medical notes.
+
+Increasingly, other countries are contributing towards and using SNOMED CT.
+
+The best ways to learn more about SNOMED CT are:
+
+* the [SNOMED International Browser](https://browser.ihtsdotools.org/)
+* the [SNOMED CT for Data Analysts](https://elearning.ihtsdotools.org/enrol/index.php?id=26) free online learning course
+* the official [SNOMED CT Starter Guide](http://snomed.org/sg), particularly chapters [5. Logical Model](http://snomed.org/sg/5.+SNOMED+CT+Logical+Model) and [6. Concept Model](http://snomed.org/sg/6.+SNOMED+CT+Concept+Model).
+
+## What is snomedizer?
+
+`snomedizer` is an R package to query and manipulate the SNOMED CT terminology.
+
+It allows you to perform the same operations as the [SNOMED International Browser](https://browser.ihtsdotools.org/).
+
+`snomedizer` is based on [Snowstorm](https://github.com/IHTSDO/snowstorm/), the official SNOMED CT Terminology Server. Snowstorm is the back-end terminology service for both the [SNOMED International Browser](https://browser.ihtsdotools.org/) and the SNOMED International Authoring Platform.
+
+# Introduction to SNOMED CT
+
+SNOMED CT contains rich knowledge about medical concepts.
+
+Try for yourself with the quick exercise below:
+
+1. Go to the [SNOMED CT International Browser](https://browser.ihtsdotools.org/).
+2. In the 'Search' tab, type in `40600002`.
+3. Click on 'Pneumococcal bronchitis' in the result list.
+4. In the right panel ('Concept Details'), click the 'Details' tab.
+5. Look for the relationships: the ontology defines that this concept is a bronchitis, and that the pathological process is infectious (non-infective bronchites exist). It also specifies that the pathogen is *Streptococcus pneumoniae*.
+6. Now click on the 'Diagram' tab. The same information is displayed graphically. Purple boxes refer to `116680003 | Is a (attribute) |` relationships, while yellow boxes notate other attributes of the concept.
```

Comment:
Reading up to this point, I am now wondering whether it would be helpful to actually start the vignette with a section about what an ontology is as I'm not sure that someone who hasn't heard about it before would grasp intuitively what a concept and what a relationship is and why these are important